.github/blog-template-demo.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: ''
+Author: Your name here
+Author handle: Any links you want associated with your name
+Date: ''
+Description: one line about your blog
+Folder with media files: ''
+Blog URL: ''
+Labels: Releases, Steering committee, Demo, Events
```

Comment:
What are these labels for?

Reply:
I think you might want to use `tags` or `categories`, though [it's not clear which to use](https://sam.hooke.me/note/2018/03/hugo-tag-and-category-pages/).